backend/src/users/users.service.ts

```diff
@@ -25,7 +25,7 @@ export class UsersService {
         return user;
     }
 
-    async update(id: number, attrs: Partial<User>) {
+    async update(id: string, attrs: Partial<User>) {
         const user = await this.findOne(id.toString());
```

Comment:
id type을 string으로 변경했는데 `toString()`이 남아있습니다.
```suggestion
        const user = await this.findOne(id);
```